lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -44,6 +46,8 @@
     private final SearchInterface searchInterface;
     private final NarrativeInfoProvider narrInfoProvider;
     private final AuthInfoProvider authInfoProvider;
+    private final static String REMOVED_GUIDS = "REMOVED_GUIDS";
```

Comment:
While the variable name should be caps, the string should follow the same style as the spec, so 'removed_guids'.